docker-compose.yml

```diff
@@ -1,5 +1,4 @@
 version: "3.4"
-
```

Comment:
Is this intended? Maybe back out this change?

Reply:
fixed in the latest commit